tests/profiler/user_workflow_fixtures.py

```diff
@@ -0,0 +1,194 @@
+from typing import List
+
+import pytest
+
+from great_expectations.core import ExpectationConfiguration, ExpectationSuite
+
+# TODO: Move these fixtures to integration tests
+
+@pytest.fixture(scope="module")
+def alice_columnar_table_single_batch():
+    profiler_config = """
+variables:
+  max_user_id: 999999999999
+  min_timestamp: 2004-10-19 10:23:54
+rules:
+  my_rule_for_user_ids:
+    domain_builder:
+      class_name: SimpleSemanticTypeColumnDomainBuilder
+      semantic_types:
+        - user_id
+    parameter_builders:
+      - name: my_min_user_id
+        class_name: MetricParameterBuilder
+        metric_name: column.min
+    expectation_configuration_builders:
+      - expectation: expect_column_values_to_be_between
+        min_value: $my_min_user_id
+        max_value: $max_user_id
+      - expectation: expect_column_values_to_not_be_null
+      - expectation: expect_column_values_to_be_of_type
+        type: INTEGER
+  my_rule_for_timestamps:
+    domain_builder:
+      class_name: SimpleColumnSuffixDomainBuilder
+      column_name_suffixes:
+        - _ts
+      user_input_list_of_domain_names:
+        - event_ts
+        - server_ts
+        - device_ts
+    parameter_builders:
+      - name: my_max_event_ts
+        class_name: MetricParameterBuilder
+        metric_name: column.max
+        metric_domain_kwargs: $event_ts.domain_kwargs
+    expectation_configuration_builders:
+      - expectation: expect_column_values_to_be_of_type
+        type: TIMESTAMP
+      - expectation: expect_column_values_to_be_increasing
```

Comment:
We have different `expectation_configuration_builders` classes, so should we not have `class_name` (for instantiation) as part of the YAML configuration?..

Reply:
See my other comments also about defaults - but here I thought particularly that we benefit from simplifying the config for simple profiler configurations by assuming `DefaultExpectationConfigurationBuilder` which just takes an expectation name and it's keyword arguments. A user can always add more specific ConfigurationBuilders (for example to bundle several expectations).

Reply:
My vote here would still be to bundle all of these under a `default` key; then other configs could be bundled under other expectation_configuration builders below that. I think the `default` key could help disambiguate a little of what's happening, though we could also set things up so that it could all just be inferred.

Reply:
@anthonyburdi @talagluck (@gilpasternak35 please see the previous comments on this topic when you have the time): I have tried what you are suggesting before -- with the same goals for a simplified configuration and getting to canonical APIs.  However, these efforts failed, because they could not mix with our prevailing configuration approach that relies on `instantiate_class_from_config()` to do what this method literally suggests.  Moreover, I had run into issues when fixing tests that were unable to disambiguate some of the configuration defaults in `base.py` with the custom settings.  Furthermore, most recently, we saw the difficulties with circumventing this approach when developing the `SimpleCheckpoint`, where the actual type does not derive from configuration as enforced by the Schema (in `base.py`).  Hence, I would be interested in a discussion on this topic in order to agree on our approach.

Reply:
FYI I added `class_name` and `module_name` for every extendable class in the verbose config. For the simplified, I think we should discuss further and added an open question to our design doc.